deploy

```diff
@@ -3,684 +3,618 @@
+            return list.payload[i].id;
+        }
+    }
+
+    let block = await createBlock('Open Growth', '10x your business!', config.keyId);
+    console.log('Created Open Growth Block:', block.payload.id);
+    return block.payload.id;
 }
 
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-// Upload Signals Handler Code
+// Create a BLOCK
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-function create_event_handler(key_id, block_id, name, event) {
+function createBlock(name, desc, keyId) {
     return http({
-        method  : 'POST'
-    ,   json    : true
-    ,   uri     : 'https://admin.pubnub.com/api/v1' + 
-                  `/blocks/key/${key_id}/`   +
-                  `event_handler`
-    ,   headers : { 'X-Session-Token' : config.login.token }
-    ,   body    : {
-            key_id : key_id
-        ,   block_id: block_id
-        ,   type: 'js'
-        ,   event: event
-        ,   channels: 'opengrowth.signals'
-        ,   name: name
-        ,   code: "console.log('hi');"
-        ,   output: 'output-0.5823105682419438'
+        method: 'POST',
+        json: true,
+        uri: 'https://admin.pubnub.com/api/v1' +
+            `/blocks/key/${keyId}/` +
+            `block`,
+        headers: {
+            'X-Session-Token': config.token
+        },
+        body: {
+            key_id: keyId,
+            name: name,
+            description: desc
         }
-    }).catch( error => {
-        console.error( 'error', error );
-    } );
+    }).catch(error => {
+        console.error('Failed to create block'.black.bgRed)
+        console.error(error.error.message.black.bgRed);
+    });
 }
 
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-// Upload Signals Handler Code
+// Get key id
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-function upload_signals_handler() {
-    return http({
-        method  : 'PUT'
-    ,   json    : true
-    ,   uri     : 'https://admin.pubnub.com/api/v1' + 
-                  `/blocks/key/${config.key_id}/`   +
-                  `event_handler/${config.signals_id}`
-    ,   headers : { 'X-Session-Token' : config.login.token }
-    ,   body    : {
-            key_id    : config.key_id
-        ,   block_id  : config.block_id
-        ,   id        : config.signals_id
-        ,   code      : config.signals_code
-        ,   channels  : 'opengrowth.signals'
-        ,   event     : 'js-before-publish'
-        ,   type      : 'js'
-        ,   log_level : 'debug'
-        ,   name      : 'Open Growth Signal Handler'
-        ,   output    : 'opengrowth.signals-info'
+async function getKeyId() {
+    let list = await listApps(config.ownerId, config.token);
+    let env = (config.env == 'Bronze') ? config.env + ' - ' + process.env.USER : config.env;
+    let id = undefined;
+
+    for (let i in list.result) {
```

Comment:
for (let app of list.result) would make references look like
app.name
instead of
list.result[i].name